Forge/src/main/java/com/blamejared/crafttweaker/platform/ForgePlatformHelper.java

```diff
@@ -158,6 +169,11 @@ public Fluid getBucketContent(BucketItem item) {
         return item.getFluid();
     }
     
+    @Override
+    public Path getGameDirectory() {
+        
+        return FMLPaths.GAMEDIR.get();
```

Comment:
Does this already work or does it need an `.absolutePath` or smth?

Reply:
The returned path is already absolute